### PR TITLE
Pin openforcefield to 0.8.0 via run_constrained

### LINKED ISSUE #7: Pin to openforcefield=0.8.0
Until https://github.com/leeping/forcebalance/issues/193 is resolved _and a new version of `forcebalance` is released_, this needs to be pinned to 0.8.0.

### PR DESCRIPTION
**Pull request by mattwthompson**
Fixes #7 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.
* [ ] ~Check logs to make sure toolkit 0.8.0 is pulled in~ won't be pulled in, but constraint applied to metadata

### CODE REVIEW

**Comment by mattwthompson** [comment]
@conda-forge-admin, please rerender

**Comment by mattwthompson** [comment]
So this pretty much copies https://github.com/conda-forge/ambertools-feedstock/pull/29/files but there does not seem to be a clear way to test this. I'll see if I can grab an artifact and test locally?

**Comment by mattwthompson** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -31,6 +31,8 @@ requirements:
     - future
     - networkx
     - lxml
+  run_constrained:
+    - openforcefield ==0.8.0
```
Comment (jaimergp) [inline recipe/meta.yaml:35]: Maybe `<=0.8.0` is better here? Or do we need 0.8.0 only?

Comment (mattwthompson) [reply recipe/meta.yaml:35]: That's probably better

**Comment by mattwthompson** [comment]
I can't conda-foo enough to be confident that this works locally - installing against the tarball does not constrain the versions of `openforcefield` that are allowed to be installed, but the constraint does propagate through to the metadata, so maybe it's fine?


```
conda search forcebalance-1.7.5-py37h9bf10a3_2.tar.bz2 --info
Loading channels: done
forcebalance 1.7.5 py37h9bf10a3_2
---------------------------------
file name   : forcebalance-1.7.5-py37h9bf10a3_2.tar.bz2
name        : forcebalance
version     : 1.7.5
build       : py37h9bf10a3_2
build number: 2
size        : 677 KB
license     : BSD-3-Clause
subdir      : osx-64
url         : file:///Users/mwt/sandbox/conda_artifacts_252042_osx_64_numpy1.16python3.7.____cpython/osx-64/forcebalance-1.7.5-py37h9bf10a3_2.tar.bz2
md5         : c64fce592f98c9e6f589efd534d53a74
timestamp   : 2020-12-14 19:45:21 UTC
constraints :
  - openforcefield ==0.8.0
dependencies:
  - future
  - libcxx >=11.0.0
  - lxml
  - networkx
  - numpy >=1.16.6,<2.0a0
  - pymbar
  - python >=3.7,<3.8.0a0
  - python_abi 3.7.* *_cp37m
  - scipy

**Comment by mattwthompson** [comment]
I think I'll merge this a few hours and test it out live. Not clear to me how else to do it and it's not too destructive, as I can just remove the `run_constrained` block and bump the build number.

**Pull request merged**